scripts/e2e.sh

```diff
@@ -133,6 +133,12 @@ echo "check metrics format with promtool"
 wget -q -O /tmp/prometheus.tar.gz https://github.com/prometheus/prometheus/releases/download/v$PROMETHEUS_VERSION/prometheus-$PROMETHEUS_VERSION.linux-amd64.tar.gz
 tar zxfv /tmp/prometheus.tar.gz -C /tmp
 cat $KUBE_STATE_METRICS_LOG_DIR/metrics | /tmp/prometheus-$PROMETHEUS_VERSION.linux-amd64/promtool check-metrics
+RESULT=$?
+if [ $RESULT -eq 1 ]; then
+    echo
```

Comment:
`echo "metrics data format is error"` should be more user friendly when this actually happens and this will help user to

Reply:
I think we need to run `set +e` before `cat $KUBE_STATE_METRICS_LOG_DIR/metrics | /tmp/prometheus-$PROMETHEUS_VERSION.linux-amd64/promtool check-metrics`. If some command line returns non-zero exit code, the `e2e.sh` will exit immediately.